src/test/kotlin/no/nav/helse/spenn/grensesnittavstemming/AvstemmingTaskTest.kt

```diff
@@ -108,9 +109,12 @@ class AvstemmingTaskTest {
         }
         assertEquals(AksjonType.AVSL, sendteMeldinger.last().aksjon.aksjonType)
 
-        assertTrue(loglog.list.filter {
-                it.level == Level.INFO && it.message.startsWith("Sender avstemmingsmeldinger med avleverendeAvstemmingId=")
-        }.isNotEmpty(), "Det skal logges avleverendeAvstemmingId på INFO-nivå")
+        val loggMeldinger = loglog.list.filter {
```

Comment:
synes blir litt for komplisert å asserter på logmeldinger, finnes det annen måte å asserte denne logikken på?

Reply:
Ser at det blir litt dumt å teste på teksten "Sender avstemmingsmeldinger med avleverendeAvstemmingId=".
Endret til å i stedet sjekke at "avleverendeAvstemmingId=$expectedValue" finnes, som jo er en bedre test og også mindre utsatt for å bryte ved uvesentlige tekst-endringer :)
Ellers tenker jeg det som en del av spec'en at det _skal_ logges disse verdiene på info-nivå slik at de er tilgjengelig i Kibana ved ev. feil, og at de som sådan bør assertes.